### ISSUE TITLE
Cannot run extension tests anymore

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#74531]
Refs: https://github.com/microsoft/vscode/issues/74436

I am on Windows and just did yo code on a typescript extension. I can run the extension but not run tests. The output I see is:

```
Error: Error: Cannot find module 'c:\Users\benjpas\Desktop\extensions-testing\end-ts\out\test'
	at I._doHandleExtensionTests (c:\Users\benjpas\AppData\Local\Programs\Microsoft VS Code Exploration\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:742:16)
	at I._handleExtensionTests (c:\Users\benjpas\AppData\Local\Programs\Microsoft VS Code Exploration\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:741:469)
	at define._startExtensionHost._readyToStartExtensionHost.wait.then.then.then (c:\Users\benjpas\AppData\Local\Programs\Microsoft VS Code Exploration\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:742:523)
	at process._tickCallback (internal/process/next_tick.js:68:7)
```

![image](https://user-images.githubusercontent.com/900690/58550309-c43c9080-820d-11e9-94a4-68ac53eab24d.png)

**Comment by bpasero** [comment]
The test runs fine from the command line via `yarn test`.

**Comment by bpasero** [comment]
Actually also seems to run fine from the sample:

![image](https://user-images.githubusercontent.com/900690/58550577-4fb62180-820e-11e9-94f2-65c2fdcae403.png)

Maybe a missing change for the generator?

**Comment by octref** [comment]
Fixed, need @aeschli to publish a new version.

**Comment by bpasero** [comment]
Thanks, I can test this by installing the generator from master, no need to publish for me.

**Comment by aeschli** [comment]
Published generator-code@1.2.0-next.1
